common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -84,7 +82,7 @@ private static String toPoolName(Class<?> poolType) {
         }
     }
 
-    public DefaultThreadFactory(String poolName, boolean daemon, int priority, ThreadGroup threadGroup) {
+    public DefaultThreadFactory(String poolName, boolean daemon, int priority) {
```

Comment:
@jasontedor yep this class is used externally and was part of releases already so we can not remove the constructor. See above :)